backend/.env.example

```
@@ -1,2 +1,5 @@
 MONGODB_URL=mongodb://localhost:27017
-DATABASE_NAME=freelancer_platform
\ No newline at end of file
+DATABASE_NAME=freelancer_platform
+JWT_SECRET=your_jwt_secret_key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_SECRET key should go before the MONGODB_URL key

(UnorderedKey)